stdlib/public/SwiftShims/LibcShims.h

```diff
@@ -112,7 +108,7 @@ static inline __swift_size_t _swift_stdlib_malloc_size(const void *ptr) {
    || defined(__HAIKU__) || defined(__FreeBSD__)
 static inline __swift_size_t _swift_stdlib_malloc_size(const void *ptr) {
 #if defined(__ANDROID__)
-#if __ANDROID_API__ >= 17
+#if !defined(__ANDROID_API__) || __ANDROID_API__ >= 17
```

Comment:
I would’ve written it as `__ANDROID_API__-17 >= 0`, but thats really equivalent.

Reply:
Let me check if that works and I can change it. Maybe with a comment because I think I will forget why I did it that way in less than 1 week.

Reply:
Seems that it doesn't work as expected. We can review this later if necessary, but I'm going to merge to try to fix the CI builds.